xamarin/Xamarin.Forms
# Fix Collection selection using SwipeView

### Description of Change ###

Fix Collection selection (CollectionView and ListView) using SwipeView.

Fixed on Android and iOS. 

### Issues Resolved ### 

- fixes #9467
- fixes #8774
- fixes #8903
- fixes #9089
- fixes #9466
- fixes #9027
- fixes #10056

### API Changes ###

 None

### Platforms Affected ### 

- iOS
- Android

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Android
![fix9466](https://user-images.githubusercontent.com/6755973/75172765-0e678980-572e-11ea-91cb-461ae047ab02.gif)

iOS
![fix9467-ios](https://user-images.githubusercontent.com/6755973/74857084-74759a80-5343-11ea-8756-efee32bade1f.gif)

Using GestureRecognizers
![77737025-764f0f80-700d-11ea-8902-91240ddf635f](https://user-images.githubusercontent.com/6755973/77759245-b4135e80-7034-11ea-96d7-647b86c9e5c0.gif)

### Testing Procedure ###
Launch Core Gallery and navigate to the SwipeView samples. Select the ListView and/or the CollectionView sample. Then, tap any item and verify that the SelectionChanged event of the CollectionView is executed.

### PR Checklist ###
